frontend/public/reducers/ols.ts

```diff
@@ -0,0 +1,14 @@
+import { RootState } from '../redux';
+
+export enum ActionType {
+  OpenOLS = 'openOLS',
+  CloseOLS = 'closeOLS',
+}
+
+type CodeBlock = {
+  id: string;
+  value: string;
+};
+
+export const getOLSCodeBlock = ({ plugins }: RootState): CodeBlock =>
```

Comment:
Technically this is a selector and not a reducer. We don't have any ols specific reducers here but since we don't have selectors in separate folders i am fine with letting this go in here. Maybe add a comment explaining where the actual reducer is and how we know about this specific selector?